shared/src/business/useCases/processStreamRecords/processWorkItemEntries.js

```diff
@@ -1,8 +1,61 @@
-const { processEntries } = require('./processEntries');
-
-exports.processWorkItemEntries = ({ applicationContext, workItemRecords }) =>
-  processEntries({
-    applicationContext,
-    recordType: 'workItemRecords',
-    records: workItemRecords,
-  });
+const { compact } = require('lodash');
+
+exports.processWorkItemEntries = async ({
+  applicationContext,
+  workItemRecords,
+}) => {
+  if (!workItemRecords.length) return;
+
+  applicationContext.logger.debug(
+    `going to index ${workItemRecords.length} work item records`,
+  );
+
+  const indexWorkItemEntry = async workItemRecord => {
```

Comment:
It appears this doesn't need to be asynchronous.

Reply:
You are absolutely correct, and I have removed it.  Thanks for catching it.